perfkitbenchmarker/providers/aws/aws_network.py

```diff
@@ -51,7 +51,7 @@ def __init__(self):
     self.firewall_set = set()
     self._lock = threading.Lock()
 
-  def AllowPort(self, vm, port):
+  def AllowPort(self, vm, port, to_port=None):
```

Comment:
Yeah, I agree. Some of the other smaller providers already implemented this so I just kept their naming scheme, but I'll go ahead and change all of them.